op-batcher/batcher/driver.go

```diff
@@ -324,6 +329,71 @@ func (l *BatchSubmitter) loop() {
 	}
 }
 
+// checkRecentTxsOnStart Check to see if there was a batcher tx sent recently that
+// still needs more block confirmations before being considered finalized
+func (l *BatchSubmitter) checkRecentTxsOnStart() {
+	if !l.Config.CheckRecentTxsOnStart {
+		return
+	}
+
+	currentBlock, err := l.Txmgr.BlockNumber(l.shutdownCtx)
+	if err != nil {
+		l.Log.Error("failed to retrieve current block number", "err", err)
+		return
+	}
+
+	currentBlockBig := new(big.Int)
```

Comment:
personal style nit: why add the type to the variable name? just seems like unnecessary overhead. also can set the uint64 on the same line to reduce a loc

Reply:
Updated in https://github.com/ethereum-optimism/optimism/pull/10193/commits/add2881fd459ad8c34fb94ffe0b70802a6f2b1b1